poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.5 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
```

Comment:
i think you need to run `make install` again to get the latest poetry version

Reply:
Unfortunately, since I have poetry installed, it will just skip over it: 

https://github.com/apache/iceberg-python/blob/325c8a55d455e479813bdfa831c450991f9c4d36/Makefile#L22-L28